theme/lumo/vaadin-grid-styles.html

```diff
@@ -134,12 +134,15 @@
       }
 
       [part~="row"][dragstart] {
-        /* Add bottom-space to the row so the drag number doesn't get clipped. Needed for IE/Edge */
-        border-bottom: 100px solid transparent;
         z-index: 100 !important;
         opacity: 0.9;
       }
 
+      #scroller[ie] [part~="row"][dragstart] {
+        /* Add bottom-space to the row so the drag number doesn't get clipped. Needed for IE */
+        border-bottom: 100px solid transparent;
```

Comment:
The extra space caused rendering issue on iOS 13 iPad pro Safari, let's just scope it to IE only. Also, evergreen Edge had changed their rendering engine so that the extra space caused the rows 100px below the row being dragged to get included in the drag image as well (scoping this to IE only is the lesser evil)